src/api.js

```diff
@@ -33,3 +33,16 @@ export const getCommentsByArticleID = (id) => {
     })
     .catch((err) => console.log(err));
 };
+
+export const patchArticle = (id) => {
+  const patchBody = {
+    inc_votes: 1,
+  };
+
+  return myApi
+    .patch(`/api/articles/${id}`, patchBody)
+    .then(({ data }) => {
+      return data.article;
+    })
+    .catch((err) => console.log(err.response));
```

Comment:
This `catch` block will break your error handling. If you catch the error here the promise will resolve with `undefined` and the catch in your component will never run. 

Take this one out